### PR TITLE
config updates

### PR DESCRIPTION
**Pull request by frostbyte73**

### CODE REVIEW

**Review by davidzhao — APPROVED** [review]
lgtm!

File: service/README.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ redis:
     username: redis username (optional)
     password: redis password (optional)
     db: redis db (optional)
+# template config
+ws_url: livekit server ws url (required if using templates)
 api_key: livekit server api key (required if using templates without supplying tokens)
```
[the remark this answers is not in the document]
Comment (frostbyte73) [reply service/README.md:41]: server doesn't hold onto any reference to the api key after request authentication, I looked into a few weeks ago it and it was really messy

[the remark this answers is not in the document]
Comment (davidzhao) [reply service/README.md:41]: I see.. just thought with your context change, it should be easier? i.e. context could carry that api key along with other details across the request lifecycle

[the remark this answers is not in the document]
Comment (frostbyte73) [reply service/README.md:41]: I can look into it again after the message bus stuff, maybe it will be easier now

**Pull request merged**